### ISSUE TITLE
Pilot Experimental Design

### ISSUE THREAD

**Issue by lenasong** [huntallcott/PhoneAddiction#1]
https://docs.google.com/document/d/1XQXFKScUl9t_bKHFfRMtbtIG9uw0PfOvdD3vHA5zUKA/edit?usp=sharing

**Comment by lenasong** [comment]
https://docs.google.com/document/d/1XQXFKScUl9t_bKHFfRMtbtIG9uw0PfOvdD3vHA5zUKA/edit?usp=sharing

**Comment by huntallcott** [comment]
@lenasong , thanks for doing this. This is great. I took a look at what you wrote and have suggested a shortened design for an initial pilot, [here](https://docs.google.com/document/d/1PtNYmkrJFR4o44Mx4Ik_OEp1TZnAsCLN3zyfey_hkeI/edit?usp=sharing) and also linked on dropbox/PhoneAddiction/ExperimentalDesign/.
I refined the questions some and gave additional precision on a possible randomization scheme. 

@gentzkow @lenasong  @KellyBuckley what do you think?

**Comment by gentzkow** [comment]
Thanks @lenasong @huntallcott. I made a bunch of comments on @lenasong's original Google doc + 1 comment in @huntallcott's modified Google doc. Let me know if you'd rather I copy those over to the Github thread here.

I definitely like the idea of the simplified pilot. I think it would be good to say for the initial round we will
1. Forget about news / politics
2. Focus only on overall usage not specific apps
3. Focus mainly on measuring WTA, demand for commitment, and changes in WTA, with well being and social interactions measured using the standard survey questions.

For the next version, we can think about various extensions, including more ambitious outcome measurement.

**Comment by lenasong** [comment]
I like the the simplified pilot too. @huntallcott @gentzkow thanks for your comments. I'm going to consolidate these two files in the next couple days and create a new document that either addresses your comments or highlight them again if it requires more thinking. @KellyBuckley thoughts? Feel free to add additional notes in the meantime.

**Comment by KellyBuckley** [comment]
@lenasong just added comments in the google doc! I haven't set up something like this before so feel free to ignore anything that doesn't make sense :)

**Comment by lenasong** [comment]
I've updated the simple pilot experimental design [here](https://docs.google.com/document/d/1-0t_pamO4v_4DWMtbz15RWAxaDf5N6h0b2dgSu3bxHM/edit#). From our discussions, there are still a few technical details to confirm and areas to brainstorm (measures of social interaction and productivity) which I've summarized in this [open questions](https://docs.google.com/document/d/1JN-EoCgx4FZLcAk9Wp8bXGrrm4kHsR7eyhF_V_TixnY/edit) file.

The survey design file is updated with a list of questions on SWB and social interactions. I went through all 4 surveys iGen used and took out the relevant questions out of their questionnaires - I'm including all things that could be somewhat influenced by phone usage and changed in a semester and therefore this is meant to be a more comprehensive list. Files including the full list of survey instruments are uploaded on our Dropbox: PhoneAddiction\Literature\iGen surveys.

The open questions file includes ideas that arise from our discussions - some technical and others that require more brainstorming - depending on the direction we want to take and whether we want to focus on measuring social interactions and/or productivity.

**Comment by huntallcott** [comment]
@lenasong I am sorry for the delay. Some thoughts are below.
1. First a procedural note: for any kind of files (data, code, notes), I think we should avoid having _v2's. Can you just overwrite my experimental design with your newer experimental design, and click "resolve" for any resolved comments, and then delete the files for your newer experimental design? Then google will keep a version history that we can refer to later. I learned this from Matt, who I think learned it from best practices.
2. On measuring pickups vs. minutes, my main concern is that pickups might not be measured well in Moment. My Moment app says I'm averaging 3.5 pickups per day, which is not accurate. Perhaps you could check yours? I would be happy to use only minutes if that's measured better.
3. For questions 7.X (SWB, time use) and exploratory questions 4.X, I think we should re-write to focus on "over the past X period of time," where X period of time refers to the length that the treatment was/will be. So for the pilot, where we would do reductions for a week, we'd re-write to ask people about SWB and time use over the past week.
4. Exploratory questions: 4.1 and 4.2. If we want these to measure substitution, which I think is sensible, then I would clarify that a “computer” means all devices other than your smartphone, 
5. Next steps: I think that a small pilot on MTurk would be great. To get there:
i. @gentzkow would comment on the revised survey.
ii. @lenasong would revise the google doc according to comments.
iii. @lenasong and/or @KellyBuckley would program the survey into Qualtrics when time allows. 
iv. @KellyBuckley would field the Qualtrics survey to say 20 MTurkers, and if it works, then maybe another 60 MTurkers soon after, using the Microsoft MTurk account.

@gentzkow what do you think?

**Comment by gentzkow** [comment]
Looks great @lenasong. I made a couple of small comments in the [Google doc](https://docs.google.com/document/d/1PtNYmkrJFR4o44Mx4Ik_OEp1TZnAsCLN3zyfey_hkeI/edit#).

One general comment is that the focus on self control in smartphone usage is going to be very very apparent to users, we we'll be especially concerned about demand effects etc.

**Comment by KellyBuckley** [comment]
I have what is probably a naive question that I'm trying to understand -- maybe this is similar to Matt's point that the focus on smartphone self control is apparent, but the survey to me feels like it is framing phone use as "bad" and face-to-face time as "good" (which I think reinforces general societal ideas about this), and I wonder how that affects the results. For example, for the question "Sometimes I lose track of time while I’m using my smartphone." I would answer yes, but I would also answer yes to the unasked "Sometimes I lose track of time when I'm with my friends." but I don't think I'm addicted to either. Or the question "During a typical week, on how many evenings do you go out for fun and recreation?" kind of implies this is a healthy thing to do, but there isn't a question asking "During a typical week, on how many evenings do you have fun social interactions with your friends via your phone?". Does it make any sense that people may end up answering the questions about how much they want to control their phone usage based on what they think they're "supposed" to do as implied by the framing, not on how much value they actually get from various apps or am I just reading too much into the questions?

**Comment by huntallcott** [comment]
@KellyBuckley I think that this kind of experimenter demand effect / social desirability bias is a serious issue. This is another reason to like "hard" outcomes like transcripts, phone usage, and willingness-to-pay. But I imagine that the plan would be to still have these questions and caveat as you suggest.
If we can think of ways of making the questions sound more neutral that's also great.

**Comment by lenasong** [comment]
@huntallcott @KellyBuckley @gentzkow Thanks for the comments. They are now either resolved and incorporated into the survey questions or responded to with a follow-up question. 

I share the same concern about demand effect. One minor change we could do is moving up the activity journal "How often do you do each of the following?" to the beginning, before our mention of Moment and use of self-control apps. Otherwise I agree with @huntallcott in using other outcomes, or eliciting some responses from peers.

**Comment by lenasong** [comment]
@KellyBuckley thanks for the additional comments.
@gentzkow @huntallcott do you have any other major changes in mind for this pilot version of the survey?

**Comment by gentzkow** [comment]
@lenasong I don't have any additional comments at this point. The most important issue seems to be the demand effects / framing one; I'd be in favor of your suggestion [here](https://github.com/huntallcott/PhoneAddiction/issues/1#issuecomment-400515967) to move up the time use question; if you and/or @KellyBuckley can see further edits to improve on this (e.g., adding some questions so we are more parallel in treatment of phone vs. face to face per @KellyBuckley's comment [here](https://github.com/huntallcott/PhoneAddiction/issues/1#issuecomment-400104326)) you should definitely feel free to suggest them.

**Comment by KellyBuckley** [comment]
I spent a lot of yesterday thinking about the framing / how to add questions to get a little more nuanced picture of how the phones are used and whether there is substitution between online interaction and face-to-face interaction when phone use is decreased. I went through the Monitoring the Future questions and have used some of them as a starting point. [Here's](https://docs.google.com/document/d/1PFdTS0WRiJZ6_OJyGlXrMBdm2rcwADCEji8gfef3uI8/edit?usp=sharing) what I came up with. Maybe it's too long but hopefully there's something useful.

**Comment by KellyBuckley** [comment]
@lenasong yes can do. just to clarify, do you want me to all the questions over or just the ones under the "cellphone questions" section?

**Comment by lenasong** [comment]
thanks @KellyBuckley! Feel free to make all changes.

**Comment by gentzkow** [comment]
@KellyBuckley, @lenasong These changes are great! I think the pilot looks really good at this point.

I made one small comment in the Google Doc.

Also, can you remind me what we have in mind for the "how many friends have you recruited" question? The design doc [here](https://docs.google.com/document/d/1XQXFKScUl9t_bKHFfRMtbtIG9uw0PfOvdD3vHA5zUKA/edit) notes the idea of having initial recruits invite friends. But I'm not sure how we're planning to implement this or what exactly the sequence of events is going to be (seems like we'll have to have some rounds of interaction with them prior to their filling out the pilot survey).

**Comment by lenasong** [comment]
@gentzkow I'm not sure how feasible it is to include the friends question for the pilot (need friends who are also on MTurk?) For the actual experiment we could do two or more rounds of subject recruitment with some kind of nominal referral bonus.

@KellyBuckley @huntallcott thoughts? did you have something else in mind for these questions?

**Comment by huntallcott** [comment]
@gentzkow, we met this morning and made some potentially final revisions to the pilot design. The discussion revolved primarily around simplification and making sure that the survey questions are geared for lessons that will help us to refine and improve the research question. @KellyBuckley will revise the questions and then start to program on Qualtrics for a 50-person pilot sometime soon.
Meeting notes [here](https://docs.google.com/document/d/1Ooq9dEyoXKExXr9ntaea6NY64GSLDE3_W3tXfuJ3ITI/edit?usp=sharing).
Let us know if any thoughts?

**Comment by gentzkow** [comment]
These notes look great to me @huntallcott. I"m all for going ahead with the 50 person pilot soon.

**Comment by KellyBuckley** [comment]
Quick update -- I'm in the process of revising the survey questions in the [google doc](https://docs.google.com/document/d/1PtNYmkrJFR4o44Mx4Ik_OEp1TZnAsCLN3zyfey_hkeI/edit?usp=sharing) based on our discussion. I'm still working on the lit review, which will change some of the questions a little but I think the general idea is there. Feel free to make updates / let me know if anything looks offtrack or missing. Will update again when done with lit review.

**Comment by lenasong** [comment]
The [latest version](https://docs.google.com/document/d/1PtNYmkrJFR4o44Mx4Ik_OEp1TZnAsCLN3zyfey_hkeI/edit#heading=h.jfklu515tbig) of survey questions look good. Here are some next steps:

1. Program the questions in Qualtrics - @KellyBuckley have you had a chance to get started on this?
2. Test feasibility of data exporting on Screen Time now, and then when available Dashboard (Google said in latest update Dashboard will be released this fall, so October according to speculations). 
3. Discuss ways we could build on our existing design for the full experiment - to deal with attrition, better measure outcomes and broaden our scope. 

My sense is that we should get started on 1 immediately and address 2 and 3 as we go along. @KellyBuckley @huntallcott @gentzkow thoughts?

**Comment by KellyBuckley** [comment]
@lenasong I haven't started putting questions in Qualtrics yet. I can start on that next week. I'd like to take one more pass through the survey questions this weekend.

**Comment by huntallcott** [comment]
Hi @lenasong @KellyBuckley @gentzkow ,
Getting caught up on this, including re-reading Lena's email from last week following up on the NY Times article. @lenasong thanks for sending that, and @KellyBuckley thanks for your work on the lit review last month.
I've also talked with a few people about this, including the LSE media studies professor Sonia Livingstone (through Nancy at MSR), Keith Chen, and Sally Sadoff.  Two quick thoughts:
1. I'm not sure where we are in terms of how we will do recruitment for the full study. After talking with Sally, I am increasingly convinced that we would be best served recruiting from say three four-year universities, ideally having psych labs, and ideally a bit diverse (say NYU, UCSB, and hopefully some lower-ranked school).
* It's possible we'll have lower attrition if we can do some surveys in person and people have a sense that they know us, and the type of students who show up at psych labs are used to taking studies and might attrit less than people we could randomly recruit online.
* I say four-year instead of two-year schools because we might want to get sophomores/juniors who have a grade history (for precision) and future (for persistence/long-term effects).
* If we are pulling transcripts, this would be easier if we have a small number of schools.

If this is still the way we want to go, I wonder if we should do our pilot at the Center for Experimental Social Science at NYU, even with summer school students right now if possible.

2. Keith Chen pointed out that it would be cool to get GPS data from subjects. We can use that for hard measures of whether people are spending more time away from their homes. We could either have our own app that gathers GPS data, or we could try to match to the GPS data that Susan Athey allegedly has at Stanford. 
* It's possible that we should abandon this idea immediately if we are worried that the treatment group would be more likely to leave their phones at home. But I'll bet that this is not a concern, and we could validate that through an endline self-report question about how many times you left home without your phone.

**Comment by huntallcott** [comment]
@lenasong -
Great, thanks. This highlights a couple of things that I feel like we don't know but would like to know.
I wonder if you could start new issue threads to report back on the following things:
1. Measuring location. 
* Deliverable is a google doc with more technical detail on if/how we can measure locations over time, either through our own app or through SafeGraph. (@gentzkow is that what Susan has?)

2. Sample recruitment for the "full" experiment at universities.
* Deliverable is a google doc with more info on options for sample recruitment. (We'll use this to back into the pilot sample.) Goal is as close to representative of US four-year college students as possible. For psych labs, how much do they cost, how large is the sample they provide, is there a chance that we'll have access at labs outside NYU. (e.g. if we call UCSB or UPenn, for example, can we get in there?), are there labs at lower-ranked schools? For non-psych labs, as a practical matter how do we recruit students? Facebook ads? (These are cheap but could be a weird sample of people.) Emails to random student lists? And if it's possible to get intuition or data on attrition rates for the different samples, that would be great. e.g. I have the intuition that people who are signed up for psych lab experiments are less likely to attrit.
Another practical question I have is whether it would be easy to pull transcripts from many schools, or whether there is in fact an advantage to sticking with a small handful.

3. Pulling data from the system apps. 
* Deliverable is a google doc with a recommendation for how this will work. It sounds like we're uncertain as to whether this needs to be in person or not, whether we need an app, etc.

In my mind the above things are higher priority than a pilot with the Moment app, because they could inform the sample and technologies that we use. Then we can come back to this thread for the pilot.

Does this sound right?
Perhaps Lena and Kelly, you can divide this up as you see fit? 
Kelly, let me know if this + Luca/Sarah is going to be too much and we can see if Raj can do more?
If we need a developer's advice and eventually possibly work and @KellyBuckley there is nobody around the office whom you know, then perhaps find somebody on Upwork and we can pay for that.

@lenasong call me if this doesn't make sense, and @gentzkow please weigh in?

**Comment by gentzkow** [comment]
Thanks all.

I am fine with exploring recruitment at universities. I agree about the advantages, and we always have the option to expand to other samples later.

On GPS: Safegraph is indeed what we have at Stanford. It is anonymous -- all you know for a given device is its sequence of GPS pings, nothing else, and the sample is too small to have much chance of finding our participants in the data even if we wanted to try. So I'd set that aside. 

We can consider writing our own app to do this. My guess, though, is that it would be quite a bit of work to make it robust. Maybe we could think about alternative measurement strategies, like an app that asks you at random times to say where you are, what you're doing, and who you're with (choosing from some coarse categories)?

**Comment by lenasong** [comment]
@huntallcott @KellyBuckley @gentzkow thanks for the input. I'll start two new threads on data collection/retrieval and sample recruitment and corresponding google docs.

**Comment by lenasong** [comment]
@huntallcott and I talked this afternoon to discuss our next steps. To get everyone on the same page, here's our timeline and components we need to work on in preparations for the pilot.

**Timeline:**
Launch the full experiment by November, start our 3-week piloting next week:
- Day 1: baseline survey
- End of week 1: midline survey including randomization 
- Week 2: screenshot upload survey only 
- End of week 3: endline survey 


**Data Collection**
Screenshots will be collected via Qualtrics surveys with links emailed to participants after an initial recruitment survey. I've been told that the file upload question type isn't yet compatible with Qualtrics SMS distribution functionality - so email is the only option for the screenshots. 

We could still use text message for other questions - such as random checking in on subjective well-being and whether the respondent is alone or with other people. Do we want to keep this option of communicating via text? We will need to set up a phone number for verification.  
**Possible Deliverable: set up phone number**

**Screen Time vs. battery use page**
iOS 12 was officially released last Monday - it's now available for download via system update on iPhones 5s and later models. I took a quick look using a friend's phone - it contains the information that's available on battery use page and more, including time spent on several main categories of apps on each day of the past week. The downside is that many people will be excluded from the survey since they haven't had the chance to update their system yet, but it seems like the right app to use for our full experiment and enough time would have passed by November for people to start using iOS12. 

If everyone's on board with this, I would suggest we try out Screen Time on our pilot, and add having iOS 12 as an additional eligibility requirement. 
**Deliverable: add iOS12 in recruitment survey**

**Qualtrics Survey** 
Thanks for coding this into Qualtrics, @KellyBuckley. Can you add instructions for transcript upload and format the WTA questions? What other information/input from everyone else do we need to finish this? 
**Deliverables: Updated survey**

**Recruitment** 
We will recruit NYU students online for the pilot, through Facebook/Google ads/other online platforms. See issue #5 for additional discussions. 
 
**Deliverables: set up recruitment on at least Google and Facebook ads so it's ready to be sent out next week.** 

**Screenshot Processing** 
Opened new issue #10 since this note is getting long. 

**Payments** 
See issue #9.

**Additional Testing before pilot**
Do we want to test out screenshots of Screen Time on a short survey similar to what we did last week, before we start our pilot? 

Any other questions or things we are unsure of that we should test out?

**Comment by huntallcott** [comment]
@lenasong thanks for getting this going. 
#### Data collection / Additional testing before pilot
I don't think we need file upload per se. What I thought we wanted was the ability for us to send an SMS as a prompt to take the screenshot, and for people to then respond to us with pictures. Can you double-check that Qualtrics doesn't allow this?
Another option is to set up our own SMS number that can receive pictures and keep track of when they were received and who sent them. Here's the kind of thing I had in mind: https://www.textmagic.com/virtual-mobile-number/.
The main tension I see is that it's best to take most of the survey on your computer instead of your phone (especially as we move into an experiment where people are being paid to not use their phones, so we don't want to force people to eat up their allotted phone time taking our survey, and you might imagine that we'd actually want everybody taking the surveys on their computers), but it's a bit cumbersome to upload screenshots from phone to computer. It could be that this is a non-issue, but it could be that this affects attrition. 
So one proposal is to have the surveys on computer, and then at the screen capture part of the survey, we say "please SMS your screenshots to 32433" or whatever number.

So it seems like we have multiple possible ways of getting these screenshots.
Iff there is any uncertainty about the easiest/lowest-attrition way to do so, then I think we should do a short survey like last week's, in which people are randomly assigned to one method vs. another.

#### Other
Otherwise I agree with the above and thank you.

Once you're at a stopping point on the pilot Qualtrics questions, can you post link to take the survey and also invite me to edit the survey? I can't find the survey in my NYU Qualtrics account.

**Comment by KellyBuckley** [comment]
@lenasong @huntallcott In regards to uploading screenshots via text: I had looked into this and found that you cannot directly respond to the text with a photo (like you could with a well-being text asking for a number or something) but you can text people a link to the survey (via Qualtrics), when clicked in the text it opens the survey in a browser on their phone, and they can directly add the screen shot there without ever needing to send the screenshot to a computer. I tested this and it worked for me.

**Comment by KellyBuckley** [comment]
@lenasong for the transcript upload question, are we asking them to download a consent form that they print, sign, and upload? Or do we have the consent form embedded and they use a digital signature?

**Comment by lenasong** [comment]
The response I received from Qualtrics support echos what @KellyBuckley said. 

I looked into textmagic, but unfortunately it is not possible to receive images since it is a SMS service (text only). I've also took a quick look at bulk MMS services (multimedia), but only saw marketing platforms and haven't found anything that fits our purposes. 

Great point raised by @huntallcott on the device participants choose to take the survey and substituting into desktops. Some thoughts on how we can address this:
-Add in language in survey that the time spent on the phone in taking the survey will not be counted
-Exclude time spent on survey using either 1) timer within qualtrics 2) time displayed on Screen Time (the app shows specific website usage from what I remember - correct me if I'm wrong)
-For the full experiment, exclude the week midline survey was taken in calculating weekly average over time

Given these, I think Qualtrics survey with reminders sent via both email and text is our best option for uploading screenshots. As @KellyBuckley mentioned, it's fairly straightforward to upload photos in the survey using a phone. We could add in a question that asks what kind of device users are using or use [meta info](https://www.qualtrics.com/support/survey-platform/survey-module/editing-questions/question-types-guide/advanced/meta-info-question/) within Qualtrics to collect this information during our pilot if we are worried about attrition due to device used.

For the transcript upload page I'm leaning towards collecting the needed information via an embedded consent, which we can then use to populate this [form](https://drive.google.com/open?id=1M8g_evDOOp96i09vWP1E8BwBTXNZXHK5) - this would be easiest for our participants. @KellyBuckley how easy will it be for us to 1) get digital signature 2) automatically populate the pdf with our survey responses later?

Otherwise, we will have to ask the respondents to leave the survey, click this [link](https://www.nyu.edu/registrar/transcript-form.html), complete the online page and make it pdf and send to us. This seems harder and could increase attrition.

On the survey itself, @KellyBuckley can you add langugae on the screenshot page to encourage manual update to iOS 12 and invite everyone to all the surveys drafted for the pilot when they are ready? Thanks!

**Comment by KellyBuckley** [comment]
@lenasong adding a question for participants to draw their signature looks easy, but programatically adding the image of the signature to the pdf looks like it would require some time to figure out how to code. Would it not be sufficient for NYU to see all the signature images and the Qualtrics question text that the signature was below (which could be the same as on the form)?

**Comment by lenasong** [comment]
@KellyBuckley I was told that any other forms will require additional approval from the office (right now I'm in touch with one person from the registrar's office, who can directly process our request without additional checks). I can see they would be hesitant to accept an aggregate excel file + zip file of signatures.

Nonetheless, I could definitely check with them whether we could send in an equivalent report that contains all the information. If Qualtrics could automatically generate a report that looks like the NYU form (responses grouped by participants, not by question), please send me a sample and I'll forward it to NYU for approval.

If we don't receive the approval, we'll just have to use their form and figure out how to populate pdf from a database.

**Comment by KellyBuckley** [comment]
Ok I've added the text about updating to iOS 12 to the recruitment survey. I also re-formatted the WTA questions in the baseline survey. And I added Hunt and Raj to both surveys.

I also added the digital signature question to the baseline survey and tested it out. Qualtrics cannot generate a report that looks like the NYU form, they can just collect the signature images. It seems like it might be possible to write code that somehow generates a pdf from a collection of images, but a little google searching wasn't enough to figure out how to do it. If the pilot is starting next week, we might need them to fill out the online form.

**Comment by KellyBuckley** [comment]
@lenasong looks like an interesting piece of software -- our computers here don't have Adobe Acrobat though so we can't run it, but maybe NYU labs have it installed? (I'd also note that it's $149 on top of having Adobe Acrobat)

**Comment by lenasong** [comment]
@KellyBuckley can you add both the embedded form and instructions for external site, and randomize between these two options?

**Comment by lenasong** [comment]
Yes, and include any other questions we still need to ask to complete the
consent form - replicating the NYU form minus fields we've already
collected.

**Comment by KellyBuckley** [comment]
@lenasong the form asks for address, which is not something we were going to originally collect and seems like unnecessary info for us. Do you know if we could submit the form without the address of the student? Otherwise we should probably update the IRB.

**Comment by lenasong** [comment]
Judging from the online form address is not required so we could leave it
out for now. I'll also double check with NYU.

**Comment by KellyBuckley** [comment]
@lenasong I updated the survey with the two transcript options and set up the Survey Flow to randomize the two blocks (there is a check box asking if they should be evenly presented, which I didn't check, should it be checked?). Take a look and see if you like the way it's set up. I also added a question asking them to upload a screenshot of the final signature page if they get the website transcript option (mostly as a mechanism to get them to do it, not that we need to review the screenshot). What do you think about that?

**Comment by lenasong** [comment]
@KellyBuckley embedded form looks great! 

For the link option - the [website](https://www.nyu.edu/registrar/transcript-form.html) actually doesn't directly send it out to NYU; it simply generates a pdf. So in place of the screenshot, we can ask them to send the pdf generated. We'll need to provide some additional instructions, ie. fill in Hunt as the recipient (no address needed) and save the pdf.

**Comment by KellyBuckley** [comment]
@lenasong got it, I updated that question in Qualtrics.

**Comment by lenasong** [comment]
I have gone through and put together our two main surveys for the pilot baseline:
1. [Recruitment survey](https://nyu.qualtrics.com/jfe/form/SV_1N8BQvxgfrCxETb) which is linked to the Google/FB/Instagram ads
2. [Baseline survey](https://nyu.qualtrics.com/jfe/form/SV_bdxuiQsF3ADqEjr
) that includes questions on screenshot upload

@KellyBuckley thanks for your help on this. 

Note the following:

- I've taken out SMS reminders altogether (since our file upload requires the user to click a link to Qualtrics anyways and I assume a college student with access to text will be able to access their inbox). 

- A question that asks for preferred email for communication is added. The initial recruitment survey asks for the participant's nyu email address as a verification step - so they will receive the baseline survey in their nyu inbox. However, subsequent communication will be done via the preferred email address. 

- For the WTA questions, we are currently asking participants to choose between no change in phone use vs. a change for the next 3 weeks (to match the duration of the pilot). An alternative question would be matching the duration of the full experiment (say 6 months). Would this be more informative for later?

- The screenshot questions are embedded in the Baseline Survey so participants have a better understanding of what to expect, and can choose to not continue participating, before we pay them $5 for participation. I'm also creating a separate survey for screenshot upload that include just these questions, to be used for subsequent weeks. Let me know if you think screenshot upload is better presented separately in a different survey (perhaps a day later after the baseline). 

-Some details on screenshot upload:
1. Screen Time gives a lot of details on Most Used apps, including specific websites. So to get the list of all apps it will take many screenshots with pages of apps that were used for less than 1 minute. For now I'm specifying that users only upload screenshots of apps used 1 minute or longer - let me know if we should increase/decrease this
2. In instructions I've tried to guide the participant through the screenshot process by asking for very specific items. The information on weekly usage can be found in one screenshot, which most users (including those who don't read instructions) can easily find and upload. What do we do with screenshot uploads that are incomplete or uploaded incorrectly otherwise (such as screenshots of apps that are not overlapping and therefore not clearly consecutive)? We could ask users to re-do the screenshot upload or simply inform them in the subsequent week. 

- I have two payment randomization in mind:
1. Fixed payment randomization: pay $5 for baseline survey, and randomize between $10 and $20 for completion of all other surveys
2. Friend invite randomization: for each friend invited/completed the study, user enters into a lottery for $1000 (with probability of winning of 1/500) or user receives $2.
if it looks good to everyone, @michael-butler1994 can you help implement these randomizations? 

@huntallcott @gentzkow @michael-butler1994 Let me know if you have any other comments/suggestions.

**Comment by michael-butler1994** [comment]
@lenasong I've filled out the surveys and they look great! My feedback is below. If it's easier to go through this on the phone, I'm free this afternoon to chat. 

Here are my notes: 

**Recruitment**
- When a user inputs email, we could include additional instructions saying: "include '@nyu.edu'"
- should we send an automatic an email to users who complete the survey?

I feel like a 1/50 chance to win $100 dollars is more palatable? Maybe?

**Baseline**
Under the compensation header on the first page:
- Language is confusing: “There is a 1 in 3 chance that you may get the opportunity to earn an additional $30 (on average),”
- Change to: "there’s a 1 in 3 chance of receiving an opportunity to earn an additional $30 dollars (on average)"
- I feel like this sentence may add confusion: “if you agree to complete an optional task proposed to you in Survey 2.”
- In the next paragraph, you write: “If you are determined to be eligible for the study”.. Haven’t we already determined that the user is eligible before sending them the baseline survey?

Under risks and benefits:
- “The research team cannot and do not guarantee or promise that you will receive any benefits from this study.  “…. why do we include that? We're paying people for this study.

On page to verify phone number:
- “receive $15 in 3 weeks.” could be changed to “receive at least $15 in 3 weeks”

On the “checking my smartphone is something” question: what’s the difference between checking the phone “automatically” vs “without thinking”?

For question on approving or disapproving friend phone behavior: I feel like “in a public space, such as movies or on a bus” could be split up into: “while on public transportation” and “while at a public event, e.g. movies or museum”

WTA Questions:
- the intro for the WTA questions  suggests the additional money is guaranteed. 
- On the WTA page, I think we could make the instructions clearer, could we substitute the word “question” for “scenario” and then label each “scenario” as scenario 1, 2, 3 and so on? The last sentence could be reworked for clarity.

**Comment by lenasong** [comment]
@michael-butler1994 Thanks for the comments. I'll give you a call shortly so we can quickly go through these together and let @huntallcott and @gentzkow know when it's updated.

**Comment by lenasong** [comment]
@michael-butler1994 and I have worked on updating the [recruitment survey](https://nyu.qualtrics.com/jfe/form/SV_1N8BQvxgfrCxETb) and [baseline survey](https://nyu.qualtrics.com/jfe/form/SV_bdxuiQsF3ADqEjr). @huntallcott @gentzkow take a look and let us know what you think?

Note that we've added these two payment randomizations:
- Fixed payment randomization: pay $5 for baseline survey, and randomize between $10 and $20 for completion of all other surveys
- Friend invite randomization: for each friend invited/completed the study, user enters into a lottery for $100 (with probability of winning of 1/50) or user receives $2
The participants will see in the baseline survey whatever payment scheme they saw in the recruitment survey. 

The timing we are currently promising is as follows: we will start recruitment via online ads asap. Participants who enter a nyu email address will receive the baseline survey by the end of this week, and the midline by end of next. 

Do we want all participants starting the survey at the same time? There's a problem with the timing that anyone who invites friends via our survey will have filled out the baseline survey, while their friends will be starting the survey later. Should we move the friend invitation question to the recruitment survey instead?

**Comment by huntallcott** [comment]
@lenasong thanks.
A. on the screening survey, it looks great except that the last question has some messed up text.
B. yes to moving friend invite to the recruitment survey.
C. The baseline survey link above doesn't work for me. it says "This survey can only be taken by invitation." I just logged into my qualtrics account, though, and so I could view it, but @gentzkow might not be able to.

**Comment by huntallcott** [comment]
1. Consent:
* “For any questions, comments or complaints, please contact Professor Matthew Gentzkow (Stanford University) or Professor Hunt Allcott (NYU) at mb7486@nyu.edu or (212) 998-8900.” Please change to “For any questions, comments or complaints, please contact Professor Hunt Allcott’s team at mb7486@nyu.edu.”
    - I think you should take Matt's name off of the consent form, just because Stanford IRB hasn’t looked at this. 
    - We need to keep that phone number off the form, as it's the econ department's number. (In the eventual revised IRB app I would just say that everybody has email and we don’t have a phone we can monitor, so a phone number is an unnecessary burden.)
   - If we are going to use this consent form we need to wait for IRB approval to launch the pilot.

2. “Are you interested in participating in future studies beyond this one? (Your answer will not affect your compensation for this study.)” This is for the main study? Cool idea. I might ask that later in the survey, though, just because at this point people don’t have a relationship with us and it looks like they are opting in to spam.

3. These are the right price discrimination questions, in my view, but I now would feel like a total jerk if we use them …

4. use_Q1 and use_Q2: these are best if we can compare them to actual data that we'll get in the screen uploads. What if we change this to, “on the average day in the past week, how many minutes do you think you had your smartphone unlocked?” and similar with pickups? 
  - Also, with pickups, it might be useful to define what that is officially as measured by screen time. Is it unlocks, or detected by motion sensor?

5. explore_Q2. “Google and Apple are both releasing built-in apps that summarize total phone usage as well as individual app usage. It will also allow users to impose app limits. How often do you plan to use or have been using this new functionality?” Not quite sure what “how often” means here. If I set a permanent limit one time, am I using it “once” or “daily”? Perhaps you can clarify this?

6. A few of these questions (use_Q3, explore_Q5) I’m not sure how I would use. Let's keep those iff you have a plan for learning from them.

7. WTA questions. Can you please re-do this as a matrix type question? That will make it faster and probably make answers more internally consistent. It should look like the payday lending survey MPL. (Michael: Surya should be able to show you that.)

8. Can you change the top of the MPL to have $5 for B instead of $0 for B? I liked the even spacing with $0 but this way we’ll have a better test of demand for commitment.

9. Are we making the pilot incentive compatible, or no? If yes, then we need to explain that we are going to randomly choose one answer from the MPL matrix and pay people based on that. The language in the payday survey (or the FB survey with the BDM) should be helpful.

10. I know I wanted to keep this pilot simple, but what do you think about adding another two MPLs, for two different categories of apps as displayed in screen time: social networking and productivity. This should be easy to add once we have the matrix down, and will give us an easy test of whether people in the pilot want. If we are incentivizing this, then we can randomly choose which of the three MPLs from which to draw the "question that counts."

11. On Q63, do you want to do categories instead of apps? That will keep the uploads simpler, would map to my proposed additional MPLs. Furthermore, there may be some people embarrassed about particular apps they use (I don’t know if there are porn apps or gaming apps or whatever), so we might get attrition here.

12. Q64. Do we need notifications? I think we do need pickups, but maybe not notifications?

13. Selective attrition. Do we want to encourage people to take the survey on their computers instead of their phones? Seems like we are mechanically incentivizing selective attrition if people feel like they want to take the survey on their phones.

**Comment by huntallcott** [comment]
also:
THIS IS TREMENDOUS!
ALL OF YOU HAVE DONE A FANTASTIC JOB!

**Comment by lenasong** [comment]
@huntallcott thanks! Here are the updated links that should also work for @gentzkow:  [recruitment survey](https://nyu.qualtrics.com/jfe/form/SV_1N8BQvxgfrCxETb) and [baseline survey](https://nyu.ca1.qualtrics.com/jfe/preview/SV_bdxuiQsF3ADqEjr?Q_SurveyVersionID=current&Q_CHL=preview).

@michael-butler1994 can you take a stab at addressing Hunt's comments and let me know if you have any questions?

**Comment by lenasong** [comment]
@huntallcott to clarify on the IRB process - should we remove the language from IRB altogether so we could launch the pilot this week? 

On incentive compatibility of pilot - I thought we wanted to elicit WTA again at midline in a week and randomizing then, so we are simply asking participants to consider the hypothetical scenarios now and get some usage data between baseline and midline. Let us know if you think we should skip the midline and just do this at the baseline for the pilot.

I'm fine with just categories. This would really cut down processing costs and make the screenshot upload exercise easy. We are losing a lot of interesting data though that we could potentially look at down the road (e.g. specific sites people visited that could be further categorized differently, instead of grouping into the browser category). @gentzkow and @michael-butler1994 any thoughts on this? 

Selective attrition of taking survey on phone vs. computer - could definitely add this, and also add language in survey that time spent on survey (as tracked by qualtrics) will be subtracted from total usage.

**Comment by huntallcott** [comment]
@lenasong:
0. IRB: can you write to them (Brenda?) just to check in and tell her that we are in a bit of a hurry?
1. I don't think we need extra usage data between baseline and midline, given that we can already collect historical usage data at baseline. So I would be in favor of doing incentivized MPLs at baseline for the pilot.
2. On categories, I think the most points we get intellectually is to show that people want to restrict themselves differently across classes of apps, and I think that the social networking category will be the key there. So I'm fine with categories at least for now. For specific apps we would aggregate them into categories in a paper anyway.
3. re: selective attrition. Good idea re: qualtrics time. Another way to do this would be to say "if you take the survey on your phone, we'll add 20 minutes to your allocation before calculating whether you hit your target." But this is only if we can observe in Qualtrics the device that people used for the survey, and only if you think it's a non-starter to get kids to take this on a real computer.

**Comment by michael-butler1994** [comment]
@lenasong I've updated the recruitment survey to include the friend referral questions, and I'm working through Hunt's feedback on the baseline survey now. I don't really know how the qualtrics logic works wrt these friend referral questions though (i.e. Display equals <>). Should I set up an automatic email to the email addresses in the friend lists, along the lines of: "Hi [friend name]! You're friend [name of original survey taker] recommended that you take this survey." I don't know if we should mention the monetary incentives in the email, to avoid appearing too spamy. It would also help to ask for the first name of the friend and the first name of the original survey taker.

**Comment by lenasong** [comment]
@huntallcott sounds good. I've followed up with Brenda on our IRB application and will keep you updated.  The device used can be tracked via [metainfo](https://www.qualtrics.com/community/discussion/547/identifying-mobile-vs-desktop-survey-completion) - @michael-butler1994, can you try adding this to the baseline survey? We should also add in the beginning of survey something along the lines of "this survey will take around 20 minutes and you may find it easier to complete on a computer."

To clarify, here's the proposed timeline for the pilot: 

- this week: recruitment
- when we reach 50 recruits: send out baseline
- one week after baseline: screenshot upload only
- two weeks after baseline: endine
 
I'm slightly worried about skipping the midline for the reasons outlined on the last page in our [pilot design](https://docs.google.com/document/d/1PtNYmkrJFR4o44Mx4Ik_OEp1TZnAsCLN3zyfey_hkeI/edit#). The issue of wanting people to have sense of usage before asking them to reduce might be relevant (and some might not have long enough history if they recently updated to iOS 12). Our current survey is arranged with WTA first, then screenshot upload. Perhaps we want to re-arrange the order if we would like to skip midline?

Here's the proposed randomization scheme for the full experiment:

> Randomization: 
> 60% randomized into C
> 39% randomized into T, meaning that we select the last MPL question on the 30 minute restriction, so everybody except the highest valuation people are randomized into the restriction.
> 1% are randomized into MPL.

Should we do something simpler (and maybe with a higher proportion of treatment) for the pilot?

@michael-butler1994 for friends referrals, I've set up embedded data for these emails (you can view them in Survey Flow) and added validation. You should be able to set up automatic emails to these email addresses, one email trigger for each friend with the condition that it's non-empty. I've also added first name. Let me know if you have any questions.

**Comment by michael-butler1994** [comment]
*disclaimer* not sure what MPL stands for, so my grammar may be off below :D  

I've revised the baseline survey, and have implemented the following:

 1. Updated consent language. 
 2.  I moved the "interested in other studies question?" to the payment block at the end of the survey
 3. You're not a jerk Hunt!
 4. It looks like a 'pickup' is measured by the number of times a user unlocks or clicks/swipes on a notification, according to this [article](https://www.theverge.com/2018/9/17/17870126/ios-12-screen-time-app-limits-downtime-features-how-to-use). Check out the language I used.
 5. explore_Q2:  I changed "a couple times" to "a couple times a year"
 6. Did not trouch use_Q3 or explore_Q_5
 7. I redesigned the WTA questions as matrix questions. I added a question on social networking, and clarified that we're counting time spent social networking on apps _and_ websites on mobile browser. The iphone screentime app is able to track social networking on browsers and apps. 
8. I changed the top of the MPL to $5 from $0.
9. I have **not** included language on the incentives for the WTA questions yet
10. As mentioned above, I added a matrix question on social networking, and productivity. Thoughts on adding an additional matrix for news? I think a lot people (including myself) spend too much time reading the news on their phone. 
11. I have not updated the language in Q63 yet. 
12. I did not remove Q64
13. Selective Attrition: I have not added language encouraging people to take the baseline survey on their computer. _Before_ someone begins filling out the baseline survey, they would be unaware of the incentivized MPLs, no? If so, we would only need to add language on survey time usage while explaining the incentivized MPLs (wherever this occurs) and at the beginning of subsequent surveys.

I'm really not sure whether we should use categories. While using categories would make the data collection process significantly easier, I think iPhone's Social Media category is way too broad. It includes not only Facebook, Instagram, snapchat, twitter, which are for social media consumption, but also iMessage, Facebook messenger, and Facetime, which are primarily for direct communication.

If we'd like to limit time spent social networking with the incentivized MPL's, I think we would rather limit time spent on Facebook rather than texting friends, no?

**Comment by huntallcott** [comment]
@lenasong @michael-butler1994 thanks very much for doing this.
1.	I’m fine with whatever timeline we can hit, and with the proportions you said, or 50-50 T-C in the pilot.
2.	Agree with @lenasong on the issue that it’s important to have a good sense of usage before asking to reduce. If you think that many people won’t have a 7-day history yet in screentime if they’ve just updated, then yes I’m fine doing incentivized elicitation only at midline, great. Then the baseline elicitation is just an unincentivized “practice.”
3.	@lenasong can you take a look at what Michael did last night and make any necessary refinements?
4. Seems like now might be a time to get midline and endline together, especially if we're still waiting on IRB.
Thanks to both of you.

**Comment by michael-butler1994** [comment]
@lenasong I set up the friend referral emails, though I couldn't figure out how to personalize them. I'll ask another RA how to personalize later today.  Sorry for the spam this morning-- I wanted to verify that the emails were BCC. If the latest friend referral email works for you, then I think the recruitment survey is ready to launch. Let me know, and I'll activate the google and facebook ads. 

I don't think I'll have much time to work on the Baseline survey today or tomorrow. We have a conference at MSR today, and I have some other work I need to get done tomorrow. I can work a bit on Saturday if you need me to though.

**Comment by lenasong** [comment]
@michael-butler1994 thanks. Were you able to add the device tracking metadata? 

I'm in a conference all day today and will take a look at the baseline survey tonight. 

The friend invitation looks good and we just received the IRB approval. I think we are ready to launch once you are able to set up personalization!

**Comment by lenasong** [comment]
One quick comment - are we confident that we are able to recruit 50 by end of the week? In the case we don't recruit enough, should we still send out the baseline by end of the week and do another round of recruitment next week, or wait until we reach 50?

**Comment by huntallcott** [comment]
Let’s see how it goes. In any event, I don’t think 50 is a hard number, and
I’d be happy with anything over 20.
-- 

Hunt Allcott

https://sites.google.com/site/allcott/home

**Comment by michael-butler1994** [comment]
@lenasong I added the metadata tracker to the baseline.

I talked to the Qualtrics expert RA in the lab. In order to send personalized emails, i.e. ("Hi Lena", instead of "Hello!"), I'll either have to write my own python script, or create 8 separate email triggers (which would be a pain to revise) and rewrite the friend referral question. Creating 8 separate email triggers would take an hour, but that's obviously not a great solution to the problem. I don't really know how long the python script would take-- I certainly couldn't get it done in the next two days. What should we do?

**Comment by lenasong** [comment]
@michael-butler1994 I took a closer look at the email trigger and made some minor edits. Take a look? If it looks good to you I say we go ahead and launch the ads now. Thanks!

**Comment by michael-butler1994** [comment]
@lenasong the ads have launched! I'll give ya'll an update in a few hours.

**Comment by lenasong** [comment]
Here are some comments on the [baseline](https://nyu.ca1.qualtrics.com/jfe/preview/SV_bdxuiQsF3ADqEjr?Q_SurveyVersionID=current&Q_CHL=preview):

- consent: we've left Matt's name (no contact info) in the form, since IRB asked us to include names of all PIs.

- I added two more MPLs so now we have 5 Multiple Price Lists: 30, 60, 120 minutes for total phone usage, and 60 min for productivity and 60 min for social networking. Let me know if you want to add or remove any of these, or perhaps reduce the number for some categories, e.g. 30 min of social media?

- @huntallcott I included notification as part of screenshot as a way to check if anyone implemented ways to reduce their usage. I've removed explore_Q_5 but kept use_Q3 - to compare what smartphone functions people value vs. spend most time on (I'm also curious how patient people are with this kind of really long, non-mandatory questions).  Happy to remove any of these if you think the survey is too long. 

- Screenshot of categories is added. Given @michael-butler1994's point on broad definition of social media, I'm keeping the section on individual apps for now, and asking instead for all apps with usage of over 30 minutes. 

- @michael-butler1994 please double check these screenshot upload questions. I'm testing with an old iPhone that's actually not my primary device so check things like if this is the right number of uploads to use. 

- Question: how should we calculate average daily usage? We won't be able to capture the exact number since there will be gaps and overlaps between weekly screenshots, so our best bet is to get an approximate that includes omissions and double-counting. Is this a problem? 

- Related to point above, perhaps we could use this to our advantage to address @huntallcott's point on selective attrition if participants use their phones instead of computer to take the survey. We could send the midline and screenshot upload surveys separately, asking for a weekly total at least 7 days after the midline survey is completed. For the endline, we could ask for the screenshot first, then do the endline survey. That way the survey time will not be part of the calculations for incentives.

**Comment by huntallcott** [comment]
@lenasong all this sounds good. 
One minor logistical point is that from an overall IRB perspective we're still safest keeping Matt off the public documents. For now he is a friend who is giving great advice. When he gets Stanford IRB approval we can promote him to PI.

**Comment by michael-butler1994** [comment]
@lenasong your instructions for the screenshots look good. We could even hyperlink to examples of the correct screenshots.

I think we should create a separate screenshot upload form. It could be the same form for baseline, midline, and endline surveys. It would prevent people from uploading screenshots through their desktop (a huge hassle). As soon as a user completes the baseline survey, we could automatically text or email them the screenshot upload form.

**Comment by lenasong** [comment]
@huntallcott done. @gentzkow let us know if you have any additional great advice for the baseline survey. We'll send it out to participants (25 so far) we recruit by then on Sunday.

@michael-butler1994 great ideas. Can you set these up, as well as an email that distributes the baseline, scheduled for Sunday afternoon/early evening?

**Comment by michael-butler1994** [comment]
Yes, setting all that up now. Should be done in a few hours. Also, I'm pretty sure people can create email accounts that appear to have "@nyu.edu"... Aside from transcript upload, do we have a method to vet these fake accounts?

**Comment by michael-butler1994** [comment]
@lenasong I've revised the baseline survey and created a separate Phone Usage Screenshot Upload Form, which you should have access to. As soon as a user finishes the baseline survey, they are emailed a link to the screenshot form.  Once they complete the screenshot form, they're supposed to receive a $5 gift card. How much of that has been set up?

Within Qualtrics, I can't find the contact list you set up with the contact list trigger, called "Pilot Recruitment'. Does that "Pilot Recruitment" list appear in your 'Contacts' tab in your Qualtrics account?

**Comment by lenasong** [comment]
@michael-butler1994 thanks. Yes I see the pilot recruitment contact list. Not sure why it's not shared among us with access to the recruitment survey. 

Did you already set up an email distribution to send out the baseline survey? If not I'll set it up for it to be sent out tomorrow, say 8 pm. Both of us are on the recruitment contact list so we'll see it when it's sent.

In terms of payment, what do you think about payment after validation of the screenshots? I can get the outsourcing companies to digitize these screenshots next week and verify these are valid screenshots and pay before they have to complete the midline survey next weekend.

**Comment by michael-butler1994** [comment]
@lenasong I have not set up an email distribution... though I created my own contact list trigger to see if it would collect all emails from the recruitment survey. It didn't work. Yeah, it would be great if you could send out the baseline survey to the recruitment contact list. 

Payment after screenshot validation seems reasonable. We would have to change the language slightly in the baseline survey and screenshot form.

**Comment by michael-butler1994** [comment]
Also, I see that you made the Baseline Survey and Screenshot forms private. Did you try testing the baseline survey after making that revision?  And if so, did you receive an email with a working link to the screenshot form?

**Comment by gentzkow** [comment]
@lenasong @huntallcott @michael-butler1994 Sorry to chime in late here!

The baseline survey generally looks good to me. A few comments. Apologies if these are answered in earlier comments I missed.

* What is the purpose of the very detailed "how do you use" questions? What will we do with the info that someone uses maps more than dating apps, say?

* What is the purpose of the "financial situation is tight" questions?

* I can't see a difference between "I do automatically" "I do without thinking" "I start doing before I realize I'm doing it" etc.

* For "Checking my smartphone is something:" and "Sometimes I lose track" -- are we sure disagree .. agree is the scale we want? Suppose I definitely agree that I sometimes do this but I also do it very rarely. What should I choose? Maybe better to ask how often these things happen? Or change prompt to read "Checking my smartphone is something I often:" / "I often lose track"?

* For "approve of the way people use their smartphones" questions: Again unclear to me how I should respond. Is the question whether I approve of how the typical person uses their phone? Or whether I sometimes / often see behavior I disapprove of? What would it mean to "strongly approve" of the way people use their phones when they are alone?

* The daily / monthly / etc. App limit question is ambiguous. It's not clear whether we're asking about setting app limits or just monitoring. Screen time on my phone gives me weekly notifications of my time use. If I plan to glance at those when they come up what should I answer? Maybe we want to specify here that we mean app limiting?

* It's odd to have Screen Time in the list of "apps" like Moment etc. since it's technically not an app but part of the OS (or part of the Settings app).

**Comment by lenasong** [comment]
@michael-butler1994 I've sent the baseline survey to both of us and the screenshot upload survey seems to work. Can you give me additional permission to view data, distribute, etc for the screenshot survey (so far I can only view the survey questions)? I'd like to confirm that data on the screenshot survey is linked to the same respondentID as baseline, or at least we can link them without having to ask participants to enter their emails again in the screenshot survey.

**Comment by lenasong** [comment]
@gentzkow thanks for the comments. 

Here are some thoughts and changes I've made:

- "how do you use" questions - this is a point @huntallcott brought up as well, and I'm happy to simplify two questions, Use_Q3 and Use_Q8 related to this issue. Question Use_Q3 asks the function that the user values the most, while Use_Q8 asks for each category, what do users think about the amount of time they are spending on it. I thought comparing these responses to actual usage gives a very rough measure of addiction (after incentives are given, do people reduce usage in apps that they think they are using too much to begin with?). Unless anyone has strong objections, I see Use_Q8 gets at this more directly and will remove Use_Q3.
- Financial situation questions are added to predict WTA. Since we won't be using them for price discrimination, should we remove them? 
- Simplifying multiple statements into one question on "Checking my smartphone is something I often do without thinking."
- Changed wording to "I often lose track"
- "approve of the way people use their smartphones" questions are derived from norms found in Hall, J. A., Baym, N. K., & Miltner, K. M. (2014). I've changed one of the options into approval/disapproval of phone usage "While they are alone on public transportation", and the question into "To what extent do you approve or disapprove of people using their phone..."
- Making Screen Time a separate question, to see if our participants are already using it at baseline
- I agree the Screen Time use question is ambiguous, so to clarify I added two questions to ask separately setting app limit and looking at app usage overview - I know we want to keep the pilot short so I promise these are the only two additions I'm making on this edit :)   

@michael-butler1994 and I have set up the personalized links for the surveys and will send them out Sunday around 8 pm. Will also close down the recruitment survey then. 

@huntallcott @gentzkow @KellyBuckley thank you again for all the suggestions and help in setting this up and let me know if you have any additional comments.

**Comment by gentzkow** [comment]
Thanks @lenasong. These changes sound good to me.

Just to be sure: I am fine keeping questions as they were if this maintains consistency with prior work / existing scales. I thought that might be the case with the "often do without thinking" question.

**Comment by lenasong** [comment]
@michael-butler1994 thanks.

@gentzkow @KellyBuckley point taken. I'm fine restoring the questions from Self-Report Behavioural Automaticity Index for the full experiment (but have to admit that I don't see much difference between the questions).

**Comment by KellyBuckley** [comment]
@lenasong [here's the background](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3552971/) on that index in particular (explains how it was constructed and why those questions are used, the paper actually covers how they were able to decrease the number of questions from 12 to 4). In general, multi-question measures of behavior require all the questions to predict the behavior since it's the correlation between the answers that leads to the predictiveness, not just the answer to one question. They do a bunch of work to figure this out and other studies use it, so I'd be hesitant to change it both for accuracy and if you want to have a comparison point to other papers. I created another issue earlier covering the different well-being scales that I'd recommend looking at if you decide to add well-being questions.

**Comment by michael-butler1994** [comment]
@lenasong I have a few clarification questions about randomly assigning people to control and treatment groups for the pilot. 
1. Will people know their treatment assignment before they fill out the midline survey? 
2. Will everyone receive the exact same midline survey?
- If the answer is Yes to 1, or No to 2, then I'll need to build a program which randomizes all Baseline survey completes into their treatment and control groups.  
- Otherwise, we could randomly assign baseline completes _or_ midline completes. I'm not sure what ya'll had in mind?

**Comment by gentzkow** [comment]
Thanks @lenasong. I'm happy to consider adding a few more questions to the midline if we think it's high value. Do you want to consult with @KellyBuckley and come up with a recommendation of what you think would be highest value?

**Comment by huntallcott** [comment]
My vote is to keep simple without the social interaction or swb questions.
We know how to do that, and I vote to keep this quick and easy.
-- 

Hunt Allcott

https://sites.google.com/site/allcott/home

**Comment by lenasong** [comment]
@huntallcott @gentzkow sounds good. @KellyBuckley let's touch base again after I'm done with the pilot logistics. Would be great to get your input when we review the well-being scales and social interactions questions. 

@michael-butler1994 I'm interested to hear how it's done in the payday/fb surveys. I had in mind randomization in the survey - when filling out the WTA questions the respondents do not know what incentives they are receiving, but at the end of the survey the randomization results are given (so no and no). We could also do randomization after the midline and send an email with result of randomization.

**Comment by michael-butler1994** [comment]
For the Facebook study, baseline completes were randomized before the midline was distributed. After respondents filled out their WTA questions on the midline (everyone received the same WTA questions/instructions), their treatment assignment was revealed in the survey. 

For the payday survey, everyone received the same survey, some sort of qualtrics randomization tool was used, and their placement was revealed at the end of the survey. Not sure if this occurred during a baseline or midline survey. 

I don't think the qualtrics randomization tool can handle the 60%-39%-1% randomization necessary for the real study, so I think randomization should occur before or after the midline. I think we should randomize baseline completes, so we can reveal during the midline survey which group they were assigned to, instead of over a follow up email. If there is significant attrition between baseline and midline though, we might not retain the 60%-39%-1% distribution we wanted, but maybe that's not a big deal.

**Comment by lenasong** [comment]
@michael-butler1994 great. Can you set up randomization as described with baseline contacts? You'll have to deleting us, incompletes, and those who said no to our consent form (they would show up as completes in the data export).

Let us know when the midline survey is ready with randomization embedded and we can all review?

**Comment by lenasong** [comment]
Here's a [preview](https://nyu.ca1.qualtrics.com/jfe/preview/SV_9pqiRafAuqv7d6l?Q_SurveyVersionID=current&Q_CHL=preview) of the midline. Thanks @michael-butler1994 for your help on this.

Note that all the participants sees the same WTA questions, but they will be compensated differently based on the randomization result (control 50%, treatment with 120 minutes 11%, treatment with 30 minutes 11%, treatment 60 minutes 8%, social media treatment 11% and productivity treatment 8%). Those who are chosen for treatment will be rewarded based on their choice on the $50 scenario (so all except highest valuation will be selected into restriction). 

@huntallcott @gentzkow let us know what you think - especially whether the instructions (copied below) on the WTA page are clear.

> In the the next section, we will present 5 different scenarios which ask you to pick Option A or Option B under certain payment conditions. Option A offers you money to use your iPhone as much as you want while Option B offers you money to use your iPhone less than [X] minutes per day on average or in a specific category over the next two weeks (anytime the phone is unlocked will count as in use). As you can see in the first scenario below, we ask you to pick Option A or B for six different payment conditions. 
> 
> Pay close attention and answer honestly, since there is a 50% chance that you will have an opportunity to receive additional rewards based on the choices you made. If you are selected for the rewards program, we will randomly choose a scenario, and offer you Option A or B for payment, based on your preferences below. You will get paid at the end of the two weeks.

We will send out one email tomorrow with $5 payment and individual feedback on everyone's screenshot from last week, and aim to send out the midline Sunday evening.

**Comment by huntallcott** [comment]
@lenasong @michael-butler1994 ,
This looks great to me. I am very excited about this.

Just confirming: the questions below are exactly the objects recorded in Screen Time?
![image](https://user-images.githubusercontent.com/23505296/47260096-c1610c80-d482-11e8-833d-d08db6559c06.png)

**Comment by lenasong** [comment]
@huntallcott yes, @michael-butler1994 reviewed this last week with definition from tech news site (I also don't see "official" definitions from Apple).

**Comment by gentzkow** [comment]
Thanks @lenasong @michael-butler1994. This looks great. 

Big question: Right now we are not prompting people to look at their actual usage on screentime before filling out the WTA questions, correct? Are we sure about that decision? If we don't, I think a lot of the variation in WTA answers will come from people being uncertain in how much they actually use their phones rather than thinking about their WTA to *reduce* usage. I.e., the answers would be quite different than if we asked people WTA to reduce their usage by 10%, 20%, etc. from the previous week's baseline.

Thoughts on whether it wouldn't be better to
1. Have people check usage on screentime then fill out the questions as they are?
2. Have people check usage on screentime then fill out the questions in terms of percent reductions rather than absolute minutes?

I also think we can work to make the Choice A vs B instructions a bit simpler / clearer.

I would suggest changing the first two paragraphs to:

> In the next section, we will present you with a series of choices between receiving an amount of money for sure) and receiving an amount of money *only if* you keep your total phone usage below a limit. The scenarios will differ in the monetary amounts and the time limit.
> 
> Pay close attention and answer honestly. After you answer all the questions, there is a 50% chance that you will be selected to actually receive the monetary rewards. If you are selected, we will randomly choose one of the scenarios for payment and assign you whichever option you chose. Your choices can therefore determine how much money you will earn and whether you will be asked to limit your phone usage.
> 
> Note: We will measure usage as the average amount of time your phone is unlocked over the next two weeks

I'd then suggest the we get rid of the text from "Scenario X..." through "...as in use" and revise the options to say, e.g.:

> $10 for sure      o o        $5 if I use my phone less than 30 minutes per day
> $10 for sure      o o        $10 if I use my phone less than 30 minutes per day
> etc.

**Comment by lenasong** [comment]
@gentzkow totally on board with the clarifications! It reads so much better now. 

Also good with changing the order of screenshot and WTA for the midline. 

Are we using the baseline WTA for anything other than practice? If not, I'm fine with the order as it is for the baseline. At the moment we have WTA as part of the main survey and screenshot sent as a follow-up survey right after. We will have to divide the baseline survey up if we were to change the order, with one section that has to be presented before screenshot to minimize experimenter demand effect.

I have a very basic question about demand estimation and would really love some clarification. I wondered if WTA should be in terms of reduction _to_ or reduction _by_, and absolute minutes vs. %. I convinced myself that it should be reduction _to_, to do demand estimation for, say 60 minutes, of screen time. Is this the right way to think about it?

**Comment by gentzkow** [comment]
Great. I'd like to hear @huntallcott's thoughts on the screenshot before vs. after and asking in minutes vs percents (i.e., 1 and 2 from my last comment).

Unless we think it's going to throw things off the rails, I think it's worth making the baseline as close as possible to what we think is our preferred version. That will make the practice more valuable. The fact that there are some technical issues in implementing the screenshot before version makes it more valuable to test it out in the baseline.

Not sure I understand your demand estimation question. Can you clarify? Maybe worth writing up a page in math to spell out what you mean?

**Comment by huntallcott** [comment]
Hi @lenasong @gentzkow ,
On @gentzkow's two issues.
1. I agree that it's important for people to have a sense of their past usage before they fill out future usage. There seems to be two ways to do this: (i) upload screenshots first. (ii) remind people of their baseline screenshots, or some average of past screenshots as we get further into the experiment, before the WTA. Either or both would be great.
2. Multiple considerations here, of course. One is what's easier to understand. The other is what fits nicely into the model. In addition to the issues you debate above, another possibility is a price per minute. I am about to open another issue thread on the model, and let's debate that there. This will take some time to resolve, so for this pilot, I don't see anything we should do other than keep things as they are.

**Comment by lenasong** [comment]
@huntallcott @gentzkow noted - we can definitely do both screenshot first and reminder of usage.

@gentzkow taking a quick glance at the framework @huntallcott proposed I think my question is no longer relevant. I was trying to understand how we can compare and aggregate answers over people with different initial usage without assuming a specific functional form.

**Comment by huntallcott** [comment]
@lenasong thanks. 
This looks good.
I would also add a couple of open-answer questions just to learn about the pilot and how it went, e.g.:
1. We are trying to make the surveys work as well as possible for future participants. Did you encounter any problems with uploading your screenshots, or any other advice you'd like to share?
2. [for those in treatment group]. How did you feel about using your phone less? How did that change other parts of your life? How did you compensate or change your behavior in order to use your phone less? 
3. Did the experiment make your more or less interested in software that could allow you to control your usage in the future? Why?
etc.

**Comment by lenasong** [comment]
@huntallcott thanks for the suggestions. I've updated the survey with these questions.

**Comment by gentzkow** [comment]
Looks good to me!

**Comment by huntallcott** [comment]
Closing this as the pilot is over
